src/script/conversation/ConversationRepository.ts

```diff
@@ -3207,27 +3199,23 @@ export class ConversationRepository {
       await this.conversationRoleRepository.updateConversationRoles(conversationEntity);
     }
 
-    if (!selfLeavingClearedConversation) {
```

Comment:
We do inject leaving a conversation event when user leaves a conversation after clearing the content beforehand (as two separate operations).